Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+ * Error Reporting: Unimplemented features are now properly reported as errors instead of being handled as if they were bugs.
```

Comment:
I searched the file for a better term, but didn't see anything. Is this fine?

Reply:
Yeah, perfectly fine.